code/_globalvars/traits.dm

```diff
@@ -5,7 +5,10 @@
 */
 GLOBAL_LIST_INIT(traits_by_type, list(
 	/mob = list(
-		"TRAIT_PACIFISM" = TRAIT_PACIFISM
+		"TRAIT_PACIFISM" = TRAIT_PACIFISM,
+		"TRAIT_WATERBREATH"	= TRAIT_WATERBREATH,
```

Comment:
Um.

Reply:
While I was adding bloodcrawl, I just decided to add waterbreath, because why not.

Reply:
but this isn't used anywhere!

Reply:
If you mean the global list here, it's used in the same file a few times and in the 
`/datum/admins/proc/modify_traits(datum/D)` proc, which presumably allows admins to add/remove traits from mobs.

Reply:
nevermind, I was thinking something else *entirely*